src/PointwiseFunctions/GeneralRelativity/WeylElectric.hpp

```diff
@@ -41,6 +41,30 @@ void weyl_electric(
     const tnsr::II<DataType, SpatialDim, Frame>&
         inverse_spatial_metric) noexcept;
 
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the scalar \f$E_{ij} E^{ij}\f$ from the electric
+ * part of the Weyl tensor \f$E_{ij}\f$.
+ *
+ * \details Computes the scalar \f$E_{ij} E^{ij}\f$ from the electric part
+ * of the Weyl tensor \f$E_{ij}\f$ and the inverse spatial metric
+ * \f$g^{ij}\f$, i.e. \f$E_{ij} = g^{ik}g^{jl}E_{ij}E_{kl}\f$.
+ *
+ * \note The electric part of the Weyl tensor needs additional terms for
+ * scalar computations in a non-vaccuum.
+ */
+template <size_t SpatialDim, typename Frame, typename DataType>
+Scalar<DataType> weyl_electric_scalar(
+    const tnsr::ii<DataType, SpatialDim, Frame>& weyl_electric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept;
+template <size_t SpatialDim, typename Frame, typename DataType>
+void weyl_electric_scalar(
+    gsl::not_null<Scalar<DataType>*> weyl_electric_scalar_part,
+    const tnsr::ii<DataType, SpatialDim, Frame>& weyl_electric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept;
+
```

Comment:
add a line `// @}`  after the function declaration (this groups the documentation for both functions)